libs/backend-apisix/src/fetcher.ts

```diff
@@ -145,15 +145,14 @@ export class Fetcher extends ADCSDK.backend.BackendEventSource {
       ),
     );
   }
-
   private listPluginMetadata() {
-    return this._list<typing.ListResponse<typing.PluginMetadata>>(
-      ADCSDK.ResourceType.PLUGIN_METADATA,
-    ).pipe<ADCSDK.Plugins>(
+    return this._list<
+      typing.ListResponse<Record<string, typing.PluginMetadata>>
+    >(ADCSDK.ResourceType.PLUGIN_METADATA).pipe<ADCSDK.Plugins>(
       map(({ list }) =>
         Object.fromEntries<ADCSDK.Plugin>(
           list.map((item) => [
-            item.key.split('/').pop(),
+            item.key.split('/').pop() ?? item.key,
```

Comment:
I'm concerned this might be incorrect, so it's best not to modify it.

The Admin API has some default rules, such as keys containing `/` being returned here.